README.md

```diff
@@ -75,7 +75,8 @@ The Bookstore is a website similar to the "Awesome Books" website. I create an s
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> You can see the live demo of this project [here](https://mrcbq.github.io/bookstore/)
+Coming soon...
```

Comment:
- [ OPTIONAL ] I think you should remove that section till you have the link of the live demo since it is not professional to have coming soon phrase instead so kindly just comment that section for now but when the live demo is ready you can add it 🙏🏻🤩🎉⭐

Reply:
OK, Thank you 👍🏼